admin/databases.sql

```diff
@@ -4,3 +4,8 @@ CREATE DATABASE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE;
 CREATE DATABASE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE_DEV;
 CREATE DATABASE IF NOT EXISTS RECOVER;
 CREATE DATABASE IF NOT EXISTS METADATA;
+DROP DATABASE IF EXISTS WORLDWIDE_ADDRESS_DATA;
+DROP DATABASE IF EXISTS SYNAPSE_DATA_WAREHOUSE_DANLU_STAGE;
+DROP DATABASE IF EXISTS US_EPA__CARBON_EMISSION_FACTORS;
```

Comment:
These are marketplace data, so you can go to the snowflake marketplace as account admin and unsubscribe (I think)

Reply:
I think you mean { Snowflake Marketplace > My requests } ? I didn't find anything so I'm not sure what there is to unsubscribe from.